core/tests/puppeteer-acceptance-tests/puppeteer-testing-utilities/test-constants.js

```diff
@@ -19,6 +19,7 @@
 let testConstants = {
   URLs: {
     home: 'http://localhost:8181/',
+    LearnerDashboard: 'http://localhost:8181/learner-dashboard',
```

Comment:
no need to add this variable now as it is not used anywhere.

Reply:
yes ! tests are running fine as long as i clear the datastore by restarting the servers.

Reply:
okay.. great!